Moonlight macOS/Moonlight_macOS.xcdatamodeld/Limelight 0.3.0.xcdatamodel/contents

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
Can we avoid duplicating these data models from iOS?

Reply:
Yes, fixed.